### PR TITLE
kata_agent: Signal our shim

### PR DESCRIPTION
**Pull request by sameo**
The Kata containers shim needs to be SIGUSR1 signalled.

See https://github.com/kata-containers/shim/pull/19

### CODE REVIEW

**Comment by sameo** [comment]
cc @sboeuf

File: shim.go [hunk 1]
```diff
@@ -128,14 +128,22 @@ func shimLogger() *logrus.Entry {
...
 }
 
-func stopShim(pid int) error {
+func signalShim(pid int, sig syscall.Signal) error {
 	if pid <= 0 {
 		return nil
 	}
 
-	shimLogger().WithField("shim-pid", pid).Info("Stopping shim")
+	shimLogger().WithField("shim-pid", pid).Infof("Signalling shim with %v", sig)
```
Comment (jodh-intel) [inline shim.go]: You could make this:

```
shimLogger().WithFields(
    logrus.Fields{
        "shim-pid" : pid,
        "shim-signal": sig,
}).Info("Signalling shim")
```

Comment (sameo) [reply shim.go]: Good point. Fixed.

**Comment by jodh-intel** [comment]
lgtm

**Pull request merged**